Backend/dev/test-tool/local-api/server.py

```
@@ -891,13 +891,19 @@ def stop_ny_rn(app: str) -> bool:
+    ".cabal-dir",
 ]
 REMOTE_DEFAULT_DIR = "/tmp/nammayatri"
 REMOTE_DEFAULT_COMMAND = (
     "cd Backend && nix develop .#backend -c , run-mobility-stack-dev"
 )
 REMOTE_BUFFER_BYTES = 256 * 1024  # last ~256 KB per session for re-attach
 
+# ── Multi-user devbox registry ──
+# Stored on the remote host so multiple developers can share a single box
+# without stomping on each other's folders or Caddy ports.
+REGISTRY_FILE = "/tmp/devbox-registry.json"
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/devbox-registry.json"

(S108)